tox.ini

```diff
@@ -31,6 +33,7 @@ deps =
     dj32: django>=3.1,<3.2
     dj40: django>=4.0,<4.1
     dj41: django>=4.1,<4.2
+    dj42: django>=4.2,<5.0
```

Comment:
This should ideally be `django>=4.2,<4.3`

Reply:
Django 4.3 isn't (and will never be) a thing, so that's the same thing...